src/Controls/tests/UITests/Tests/Concepts/AlertsGalleryTests.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Maui.Controls.Sample;
+using NUnit.Framework;
+using OpenQA.Selenium.Appium;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests
+{
+	public class AlertsGalleryTests : CoreGalleryBasePageTest
+	{
+		public AlertsGalleryTests(TestDevice device)
+			: base(device)
+		{
+		}
+
+		protected override void NavigateToGallery()
+		{
+			App.NavigateToGallery("Alerts Gallery");
+		}
+
+		[Test]
+		public void AlertCancel()
+		{
+			if (Device == TestDevice.Windows)
+				Assert.Ignore("UI testing alert code is not yet implemented on Windows.");
```

Comment:
Ignoring Windows for now. I probably will implement it sometime in the new year - or if someone gets to it first.